src/views/CustomTileDialog/ImportExport/index.tsx

```
@@ -1,107 +1,164 @@
+            // Find the matching group data
+            const userData = gameModeGroups.find(
+              (entry) => entry?.intensity === Number(intensity) && entry?.value === group
+            );
 
-        return actionText;
-      }
-    );
+            let actionText = '';
+            actionText += `[${userData?.group || group} - ${userData?.translatedIntensity || intensity}]\n`;
+            actionText += action;
+            actionText += tags?.length ? `\nTags: ` + tags?.join(', ') : '';
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)